### DISCUSSION TITLE
Efficient_Ad downloading and running on Imagenette

### DISCUSSION THREAD

**Discussion by ghost** [open-edge-platform/anomalib#1468] [Q&A] [answered]
I am running anomalib's efficient_ad after modifying the config file with the custom dataset just like the way I did this for other models in anomalib. I used the following command to run it:

python tools/train.py --model efficient_ad --config /path_to_config/stfpm_TCL_config.yaml

The config file is modified as follows for the custom dataset:
![image](https://github.com/openvinotoolkit/anomalib/assets/63196905/a74ab726-ac43-48a2-84c7-6ddc8b3bdb94)

When I run, it first starts downloading Imagenette2 (more than 1 GB). What is wrong?

**Comment by abc-125** [comment, marked as answer]
> it first starts downloading Imagenette2

EfficientAD uses the ImageNet dataset (or Imagenette2) during the training to penalize the student for imitating the teacher for out-of-distribution images, so downloading this dataset is an expected behavior. 

> Then, runs inference on that instead of my custom dataset. 

Can you please provide more details on this? Which version of the library are you using, logs, etc.

**Reply by ghost** [reply]
Thanks for clarifying the first point. I am sorry I was mistaken in the second point of inference on Imagenetter2. It was just taking a lot of time during the training but eventually was trained successfully on my custom dataset. I have edited the question to remove this second point.
